src/SourceCode.Clay.OpenApi/Builders/OperationBuilder.cs

```diff
@@ -90,6 +90,12 @@ public class OperationBuilder
         /// </summary>
         public OperationBuilder()
         {
+            Tags = new List<string>();
+            Parameters = new Dictionary<ParameterKey, Referable<Parameter>>();
+            Responses = new Dictionary<ResponseKey, Referable<Response>>();
+            Callbacks = new Dictionary<string, Referable<Callback>>();
```

Comment:
StringComparer.Ordinal

Reply:
I will have to write a comparer based on the spec. This will be updated as part of #80.